src/routes/notifications.ts

```diff
@@ -0,0 +1,49 @@
+import { NextFunction, Request, Response, Router } from "express";
+import * as _ from 'lodash';
+
+import { NotificationService } from "../notification-service";
+import { HTTP } from "./http";
+
+export class NotificationsRoute {
+
+  public static create(router: Router, notificationService: NotificationService) {
+
+    let notificationsRoute = new NotificationsRoute(notificationService);
+    let methodsToBind = ['requestSubscription']
+    _.bindAll(notificationsRoute, methodsToBind);
+
+    router.post('/', notificationsRoute.requestSubscription);
+  }
+
+  constructor(private notificationService: NotificationService) { }
+
+  requestSubscription(req: Request, res: Response, next: NextFunction) {
+    const notificationRequest: PushSubscriptionJSON = req.body;
+
+    if (!notificationRequest) {
+      return res.status(HTTP.BAD_REQUEST).send('Invalid data: Notification Object is mandatory');
+    }
+
+    if (
+      !notificationRequest.endpoint
+      || !notificationRequest.keys
+      || !notificationRequest.keys.auth
+      || !notificationRequest.keys.p256dh
+    ) {
+      return res.status(HTTP.UNPROCESSABLE_ENTITY).send('Malformed Notification Object');
+    }
+
+    this.notificationService.addSubscription(notificationRequest);
+    res.status(HTTP.OK).send({ message: 'successfully subscribed' });
+
+    const notificationPayload = {
+      title: '🐵 Push-Benachrichtigungen aktiv ✉️',
```

Comment:
Unknown character here, is it just my OS?

Reply:
Ich sehe nen Affen und nem Brief

Reply:
Aaah, das ist ein Brief! Sieht bei mir aus wie ein durchgestrichenes Kästchen für ein unbekanntes Zeichen